packages/bot/src/bot.ts

```diff
@@ -144,6 +141,7 @@ export interface EventHandlers {
   automodActionExecution: (payload: AutoModerationActionExecution) => unknown
   threadCreate: (thread: Channel) => unknown
   threadDelete: (thread: Channel) => unknown
+  threadListSync: (payload: { guildId: bigint; channelIds: bigint[] | unknown; threads: Channel[]; members: ThreadMember[] }) => unknown
```

Comment:
just one small thing
```suggestion
  threadListSync: (payload: { guildId: bigint; channelIds?: bigint[]; threads: Channel[]; members: ThreadMember[] }) => unknown
```

Reply:
oh , ye that actually much better